src/com/amazon/corretto/crypto/provider/AmazonCorrettoCryptoProvider.java

```diff
@@ -266,15 +266,16 @@ public AmazonCorrettoCryptoProvider() {
 
         Utils.optionsFromProperty(ExtraCheck.class, extraChecks, "extrachecks");
 
-        if (!Loader.IS_AVAILABLE && DebugFlag.VERBOSELOGS.isEnabled()) {
```

Comment:
good catch. was this a bug? i.e. we wouldn't have returned early if loader was unavailable but verbose logging hadn't been enabled?

Reply:
Yes, this was a bug that I ran into when debugging partially completed versions of this PR that did not load the native libraries correctly. In practice this hasn't been an issue since ACCP was always loading native libraries correctly.